modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/view_settings.jsp

```diff
@@ -14,6 +14,26 @@ PublicationsConfigurationDisplayContext publicationsConfigurationDisplayContext
 
 renderResponse.setTitle(LanguageUtil.get(request, "settings"));
 
+Portal portal = PortalUtil.getPortal();
+
+String languageOverridePortletLink = PortletURLBuilder.create(
+	portal.getControlPanelPortletURL(request, "com_liferay_portal_language_override_web_internal_portlet_PLOPortlet", PortletRequest.RENDER_PHASE)
+).setRedirect(
+	ParamUtil.getString(request, "backURL", portal.getCurrentCompleteURL(request))
```

Comment:
We can use the `currentURL` variable that is already available in the jsp. It's included with the `<liferay-frontend:defineObjects />` tag in our init.jsp. You can see the list of variables added in [DefineObjectsTag.java](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/DefineObjectsTag.java#L53-L54).